boot/LoadLinux.c

```diff
@@ -172,6 +172,9 @@ int LoadKernelNative(CONFIGENTRY *config) {
 	
 	I2CTransmitWord(0x10, 0x0c01); // Close DVD tray
 	
+	VIDEO_ATTR=0xffd8d8d8;
```

Comment:
I hope that we'll eventually move this into a function like `VideoSetPrintColor` or something (I'm not sure what this does to be honest).

Someone should create an issue about this. Because `VIDEO_ATTR` is not very self-documenting.

Reply:
This variable changes the color of printed text. It could be probably done as a macro, but it's something for later.

Reply:
Consider creating an issue about this (which documents what this does, and makes people aware that we'd be interested in refactoring this code)